code/__DEFINES/botany.dm

```diff
@@ -7,3 +7,6 @@
 //Both available scanning modes for the plant analyzer.
 #define PLANT_SCANMODE_STATS		0
 #define PLANT_SCANMODE_CHEMICALS 	1
+
+//Seed flags.
+#define NO_MUTATE_EARLY	(1<<0)
```

Comment:
Don't do this, avoid negative variables. Not to end up with double-negative stuff like `if(!no_stuff)` which is much more clear as a `if(stuff)`